tests/float-tests.lisp

```diff
@@ -99,7 +99,7 @@
   (declare float-checklist ((math:Dividable Integer :a) => (List :a)))
   (define float-checklist
     (coalton-prelude:zipWith
-     math:general/ (coalton:the (coalton:List coalton:Integer) (coalton-prelude:range -100 100)) (coalton-prelude:range 200 1))))
+     math:general/ (coalton:the (coalton:List coalton:Integer) (coalton-prelude:range -100 99)) (coalton-prelude:range 200 1))))
```

Comment:
?

Reply:
This doesn't change behavior (since `zipWith` stops once it reaches the end of either iterator), but just makes more explicit that the zipped list goes from (-100 200) to (99 1).
(I had just stumbled on this when trying to debug the test, which ended up just being a problem with Windows and libraries.)